packages/expo-av/src/Video.tsx

```diff
@@ -143,53 +122,55 @@ export default class Video extends React.Component<VideoProps, VideoState> imple
     return status;
   };
 
-  // ### iOS Fullscreening API ###
+  // Fullscreening API
 
-  _setFullscreen = async (value: boolean) => {
+  _setFullscreen = async (value: boolean): Promise<AVPlaybackStatus> => {
     return this._performOperationAndHandleStatusAsync((tag: number) =>
       ExpoVideoViewManager.setFullscreen(tag, value)
     );
   };
 
-  presentFullscreenPlayer = async () => {
+  /**
+   * This presents a fullscreen view of your video component on top of your app's UI. Note that even if `useNativeControls` is set to `false`,
+   * native controls will be visible in fullscreen mode.
+   * @return A `Promise` that is fulfilled with the `AVPlaybackStatus` of the video once the fullscreen player has finished presenting,
+   * or rejects if there was an error, or if this was called on an Android device.
+   */
+  presentFullscreenPlayer = async (): Promise<AVPlaybackStatus> => {
     return this._setFullscreen(true);
   };
 
-  presentIOSFullscreenPlayer = () => {
-    console.warn(
-      "You're using `presentIOSFullscreenPlayer`. Please migrate your code to use `presentFullscreenPlayer` instead."
-    );
-    return this.presentFullscreenPlayer();
-  };
-
-  presentFullscreenPlayerAsync = async () => {
+  /**
+   * @hidden note(simek): this method is not used or mentioned anywhere, let's hide it for now.
+   */
+  presentFullscreenPlayerAsync = async (): Promise<AVPlaybackStatus> => {
```

Comment:
Let's remove it altogether, shall we? 🤔 
If we decide to migrate the methods to the ones with `Async` suffix we'll do it and restore the behaviour.
Let's add a note about this in CHANGELOG file just in case.

Reply:
I'm going to removed this method then. I was considering adding deprecation warning and keeping this in at least for one SDK release, but I think it would add more confusion than just removing this method. We already have few other breaking changes in this changeset, so at this point of time I think it's fine.